content/docs/other-guides/freq-ask-questions.md

```diff
@@ -0,0 +1,12 @@
++++
+title = "Frequently asked questions"
+description = ""
+weight = 110
++++
+
+
+### Does Kubeflow Support Helm?
+
+Yes Kubeflow support Helm. You don't need to use ksonnet to use TFServing, TFJob and others. You can define those resources using whatever K8s tooling you want; raw YAML, Helm, Kustomize, Ksonnet.
+
+We provide Kustomize mostly as convenient syntax sugar. The arena CLI might have Helm Charts (https://github.com/helm/charts) if that's the route you prefer.
```

Comment:
- We provide Kustomize? This seems to lack context, considering the question is about Helm. This might need more explanation. "Provide" might also not be the best  possible word choice.
- Check capitalization for Arena.
- `Helm Charts` > `Helm charts`.
- The CLI might have Helm charts? Why might?
- Markdown links use the following format: `[link text](link)`. Take another shot at linking "Helm charts".

Reply:
To my knowledge the helm charts aren't anywhere up to date. So lets not point people at that.

Reply:
@jlewi I have updated page, Would you please take a look.